script/.electron.attributes

```diff
@@ -0,0 +1 @@
+*.mm diff=objc
```

Comment:
```suggestion
*.mm diff=objc
*.m diff=objc
```

per the `gitattributes` file in https://opensource.apple.com/tarballs/Git/Git-101.1.tar.gz